test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -355,16 +355,13 @@ def __iter__(self):
         return iter(self.orderedlist)
 
     def host_keys(self):
-        return map(lambda h: h.split('@')[0], self.keys())
+        return [h.split('@')[0] for h in list(self.keys())]
 
-    def keys(self):
-        return self.orderedlist
-
-    def itervalues(self):
-        return (self[key] for key in self.orderedlist)
+    '''def keys(self):
```

Comment:
why commented out? also why itervalues() removed?

Reply:
This is a mistake on my part. I missed a change. We commented out keys because we were getting an error in PBSServiceInstanceWrapper. After the fix, I missed to un-comment it.

As for itervalues(), it's deprecated in Python 3.
[nehap@neha20 pbspro]$ python2
Python 2.7.5 (default, Jun 20 2019, 20:27:34) 
[GCC 4.8.5 20150623 (Red Hat 4.8.5-36)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> dir(dict)
['__class__', '__cmp__', '__contains__', '__delattr__', '__delitem__', '__doc__', '__eq__', '__format__', '__ge__', '__getattribute__', '__getitem__', '__gt__', '__hash__', '__init__', '__iter__', '__le__', '__len__', '__lt__', '__ne__', '__new__', '__reduce__', '__reduce_ex__', '__repr__', '__setattr__', '__setitem__', '__sizeof__', '__str__', '__subclasshook__', 'clear', 'copy', 'fromkeys', 'get', 'has_key', 'items', 'iteritems', 'iterkeys', 'itervalues', 'keys', 'pop', 'popitem', 'setdefault', 'update', 'values', 'viewitems', 'viewkeys', 'viewvalues']
>>> 
[nehap@neha20 pbspro]$ python3
Python 3.6.8 (default, Apr 25 2019, 21:02:35) 
[GCC 4.8.5 20150623 (Red Hat 4.8.5-36)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> dir(dict)
['__class__', '__contains__', '__delattr__', '__delitem__', '__dir__', '__doc__', '__eq__', '__format__', '__ge__', '__getattribute__', '__getitem__', '__gt__', '__hash__', '__init__', '__init_subclass__', '__iter__', '__le__', '__len__', '__lt__', '__ne__', '__new__', '__reduce__', '__reduce_ex__', '__repr__', '__setattr__', '__setitem__', '__sizeof__', '__str__', '__subclasshook__', 'clear', 'copy', 'fromkeys', 'get', 'items', 'keys', 'pop', 'popitem', 'setdefault', 'update', 'values']
>>>